### ISSUE TITLE
op_shortcode_template always points to plugin dir

### ISSUE THREAD

**Issue by pnpetroff** [janizde/WP-Opening-Hours#96]
It seems that using the `op_shortcode_template` filter is impossible to be used in a theme.

You can't create a custom template file the theme directory as it always points to the plugin one:

```php
// wp-opening-hours\classes\OpeningHours\Module\Shortcode\AbstractShortcode.php

 $templatePath = apply_filters(self::FILTER_TEMPLATE, $templatePath, $this);
 $templatePath = sprintf('%s/views/%s', op_plugin_path(), $templatePath);

 $view = new ViewRenderer($templatePath, $data);
```

`op_plugin_path()` is always appended to the URL.

Shouldn't the apply filter be after the `op_plugin_path()` line?

**Comment by janizde** [comment]
Yes that's right, would be better to have it without constraints. I currently don't have much time for the plugin, so send a pull request if you'd like. If so, then please add an additional filter, as other people might use the FILTER_TEMPLATE under the assumption that it will be appended to /views and maybe mark the old one as deprecated. (Would be great to have this change reflected in the filter docs).

Cheers and thanks for reporting
